templates/gitea.ini.j2

```diff
@@ -58,8 +58,10 @@ APP_DATA_PATH         = {{ gitea_home }}/data
 LFS_START_SERVER      = true
 ; Where to store LFS files.
 LFS_CONTENT_PATH      = {{ gitea_lfs_content_path }}
+; if the LFS sotre is not offline/local
+LFS_JWT_SECRET   = {{ gitea_lfs_secret }} 
```

Comment:
trailing whitespace

Reply:
i checked w/ ansible-linter .... there whitespaces in my code o.O